src/olympia/activity/tests/test_commands.py

```diff
@@ -67,3 +68,36 @@ def test_with_token_and_version_ignores_version(self):
     def test_no_tokens_no_version_is_error(self):
         with pytest.raises(CommandError):
             call_command('repudiate_token')
+
+
+def test_backfill_ratinglog_command():
+    user = user_factory()
+    addon = addon_factory()
+
+    no_rating_in_args_log = ActivityLog.create(amo.LOG.ADD_RATING, user=user)
+    assert no_rating_in_args_log.arguments == []
+
+    missing = Rating.objects.create(addon=addon, user=user)
+    missing_log = ActivityLog.create(amo.LOG.ADD_RATING, user=user)
+    missing_log.set_arguments((missing,))
+    missing_log.save()
+
+    has_rating = Rating.objects.create(addon=addon, user=user)
+    has_rating_log = ActivityLog.create(amo.LOG.ADD_RATING, has_rating, user=user)
+    assert has_rating_log.arguments == [has_rating]
+
+    other_action = Rating.objects.create(addon=addon, user=user)
+    other_action_log = ActivityLog.create(amo.LOG.CHANGE_STATUS, addon, user=user)
+    assert other_action_log.arguments == [addon]
+    other_action_log.set_arguments((addon, other_action))
+    other_action_log.save()
+
+    assert RatingLog.objects.count() == 1
+    call_command('backfill_ratinglog')
+
+    assert RatingLog.objects.count() == 2
+    ratinglog1, ratinglog2 = list(RatingLog.objects.all())
```

Comment:
Add a `.order_by('-id')` to guarantee ordering (default ordering on `created` might lead to intermittent failures if both are created at the exact same time - it's unlikely in this scenario but you never know)